src/Altinn.AccessManagement.Core/Services/AuthorizedPartiesService.cs

```diff
@@ -285,6 +293,26 @@ private async Task<List<AuthorizedParty>> BuildAuthorizedParties(int subjectUser
             authorizedParty.EnrichWithResourceAccess(delegation.ResourceId);
         }
 
+        var instanceDelegations = await GetInstanceDelegations(subjectUserId, subjectPartyIds, cancellationToken);
+        var instanceParties = await _contextRetrievalService.GetPartiesByUuids(instanceDelegations.Select(i => i.FromUuid), false, cancellationToken);
+        foreach (var delegation in instanceDelegations)
+        {
+            if (instanceParties.TryGetValue(delegation.FromUuid.ToString(), out var instanceParty))
+            {
+                throw new UnreachableException($"Get AuthorizedParties failed to lookup party with uuid {delegation.FromUuid} while building instance delegations list");
+            }
+
+            if (authorizedPartyDict.TryGetValue(instanceParty.PartyId, out var authorizedParty))
+            {
+                continue;
+            }
+
+            authorizedParty = new AuthorizedParty(instanceParty);
+            authorizedParty.AuthorizedInstances.Add(delegation.InstanceId);
```

Comment:
Her må vi vel kanskje finne på noe lurt.. Vi har ingen garanti for at instanceId er unik på tvers av ressurser. Så AuthorizedInstances vi gir ut må nok inkludere både ressurs og instans..

Reply:
returnerer en liste med objeketer istedenfor strings
{
  resourceId: "<id>"
  instanceId:
}